views/refresh.html

```diff
@@ -45,6 +45,6 @@
       </div>
     </section>
 
-    <script src="/{{ localeDir }}/error/scripts/main.js"></script>
+    <script src="/scripts/error.js"></script>
```

Comment:
this should be `/resources/scripts/error.js`

Reply:
Thanks, fixed.